packages/in_app_purchase/in_app_purchase/lib/src/in_app_purchase/product_details.dart

```diff
@@ -33,6 +35,13 @@ class ProductDetails {
   /// Formatted with currency symbol ("$0.99").
   final String price;
 
+  /// The unformatted price of the product, specified in the App Store Connect or Sku in Google Play console based on the platform.
+  final double rawPrice;
```

Comment:
What is the base unit of this rawPrice? For example, if the currency code is USD, is the base unit dollar, or cents? We also need to figure out how we can document this.

Reply:
The base unit is the other `currencyCode` property that was added in this PR. Outside of the context of this PR it is indeed not very clear. I've made some changes in which I tried describing it clearly. Please let me know if it is sufficient.